src/TourManager/TourManager.Api/Bootstrapper/AutoMapperConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace TourManager.Api.Bootstrapper
+{
+    public static class AutoMapperConfiguration
+    {
+        public static IServiceCollection AddAutoMapper(this IServiceCollection services)
+        {
+            var mapperConfig = new MapperConfiguration(mc =>
```

Comment:
Use `AutoMapper.Extensions.Microsoft.DependencyInjection` instead of using custom config 
https://www.codementor.io/@zedotech/how-to-using-automapper-on-asp-net-core-3-0-via-dependencyinjection-zq497lzsq